analyses/tmb-vaf-longitudinal/util/function-create-corplot.R

```diff
@@ -0,0 +1,66 @@
+#' Create corplots 
+#'
+#' @param maf 
+#' @param timepoints_other_plot 
+#' @param timepoints_deceased_plot 
+#' @param sid 
+#'
+#' @return A data frame with each possible combination of timepoints and create VAF corplot
+#' @export
+#'
+#' @examples
+create_corplot <- function(maf, timepoints_other_plot, timepoints_deceased_plot, sid) {
+
+  # Split maf, create df, and rename VAF column based on time point
+  timepoint_df <- maf[which(maf$timepoints_other == timepoints_other_plot), ] %>%
+    select(gene_protein, Hugo_Symbol, VAF) 
+  colnames(timepoint_df)[colnames(timepoint_df) == "VAF"] <- timepoints_other_plot
+
+  deceased_df <- maf[which(maf$timepoints_deceased == timepoints_deceased_plot), ] %>%
+    select(gene_protein, Hugo_Symbol, VAF)
+  colnames(deceased_df)[colnames(deceased_df) == "VAF"] <- timepoints_deceased_plot
```

Comment:
```suggestion
  colnames(deceased_df)["VAF"] <- timepoints_deceased_plot
```
Can just call the colname without filtering

Reply:
Same here de85a071d0572e03bc72085daab17aa570ac82b8